main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -55,6 +55,75 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     );
   };
 
+  function successCallBack(columnName,dismissDialog)
+  {
+      var serviceUrl = null;
+      var service = null;
+      if (column.reconConfig) {
+        serviceUrl = column.reconConfig.service;
+      }
+      if (serviceUrl) {
+        service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+      }
+      if (service && service.view){
+        Refine.postCoreProcess
+        (
+        "add-column",
+        {
+        baseColumnName: column.name,
+        newColumnName: columnName,
```

Comment:
The indentation can still be improved here. You can look for other calls to `Refine.postCoreProcess` for inspiration :)